scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -74,16 +74,31 @@
   <div class="panel-group">
     {% for institute, panels in panel_groups %}
       <div class="panel panel-default">
-        <div class="panel-heading">{{ institute.display_name }} - Panels</div>
-        <ul class="list-group">
+          <div class="panel-heading">{{ institute.display_name }} - Panels</div>	  
           {% for panel in panels %}
-            <li class="list-group-item">
-              <a href="{{ url_for('panels.panel', panel_id=panel._id) }}">
-                {{ panel.display_name }} ({{ panel.version }}, {{ panel.date.date() }})
-              </a>
-            </li>
+            <div class="panel-group" id="accordion-{{ panel.panel_name }}">
+	      <div class="panel panel-default">
+		<a href="{{ url_for('panels.panel', panel_id=panel._id) }}">
+                  {{ panel.display_name }} ({{ panel.version }}, {{ panel.date.date() }}, {{ panel.genes|length }} genes)
```

Comment:
Ok so I'm testing this and get a bit annoyed with this "header" line. Could you use only `display_name` here?
Otherwise the information is repeated and things get a little confusing

Reply:
The other things looks and works great!

Reply:
Yep, agree - that's redundant.